package.json

```diff
@@ -32,7 +32,7 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-component-footer": "10.3.0",
     "@edx/frontend-component-header": "2.6.1",
-    "@edx/frontend-platform": "1.15.6",
+    "@edx/frontend-platform": "2.6.2",
```

Comment:
We're actually [on 2.6.2 now](https://github.com/openedx/frontend-app-profile/blob/master/package.json#L35), I'm not sure why this diff still shows.